From ec00f41ec053792bbf8bde6255ad403eec966f61 Mon Sep 17 00:00:00 2001
From: Honore Tricot <honorex.tricot@intel.com>
Date: Fri, 7 Sep 2012 16:06:05 +0200
Subject: [PATCH 22/78] WLAN: SleepPolicy / Periodic scan integration

BZ: 53958

When Periodic scan is enable and with a lot of
SSID to scan, the 2 series of probe request
are not sent as expected.
Decrease the sched_scan / .dwell_time_delta_per_probe
permits to send the 2*16 probe (worst case)

Change-Id: Ide338117a9cc33f768a346602f421df3e12e0503
Signed-off-by: Honore Tricot <honorex.tricot@intel.com>
Reviewed-on: http://android.intel.com:8080/64853
Reviewed-by: Trivelly, Jean <jean.trivelly@intel.com>
Reviewed-by: Lemetayer, Jean-marieX <jean-mariex.lemetayer@intel.com>
Reviewed-by: Regairaz, Julie <julie.regairaz@intel.com>
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77979
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wl12xx-compat/drivers/net/wireless/wl12xx/main.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/wl12xx-compat/drivers/net/wireless/wl12xx/main.c b/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
index 44f71f4..74abd93 100644
--- a/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
+++ b/wl12xx-compat/drivers/net/wireless/wl12xx/main.c
@@ -289,7 +289,7 @@ static struct conf_drv_settings default_conf = {
 		.base_dwell_time              = 7500,
 		.max_dwell_time_delta         = 22500,
 		/* based on 250bits per probe @1Mbps */
-		.dwell_time_delta_per_probe   = 2000,
+		.dwell_time_delta_per_probe   = 1500,
 		/* based on 250bits per probe @6Mbps (plus a bit more) */
 		.dwell_time_delta_per_probe_5 = 350,
 		.dwell_time_passive           = 100000,
-- 
1.7.9

